rasa/cli/test.py

```diff
@@ -100,6 +100,10 @@ def test_nlu(args: argparse.Namespace) -> None:
 
     nlu_data = cli_utils.get_validated_path(args.nlu, "nlu", DEFAULT_DATA_PATH)
     nlu_data = data.get_nlu_directory(nlu_data)
+    output = args.out or DEFAULT_RESULTS_PATH
+
+    if not os.path.exists(output):
```

Comment:
You can use 
```
    io_utils.create_directory(output)
```

Reply:
The same command was already used in the test_core function of that file. Would you like me to update that too?

Reply:
Sure. Thanks!